portfolio/src/main/webapp/style.css

```diff
@@ -13,6 +13,57 @@
   font-weight: bold;
 }
 
+#experience {
+  margin-top: 100px;
+}
+
+.timeline {
+  list-style-type: none;
+  padding-left: 0px;
+  position: relative;
+}
+
+.timeline::before {
+  content: ' ';
+  background: #d4d9df;
+  display: inline-block;
+  position: absolute;
+  left: 30px;
+  width: 3px;
+  height: 100%;
+  z-index: 10;
+}
+
+.timeline > li {
+  margin: 20px;
+  padding-left: 50px;
+}
+
+.timeline > li::before {
```

Comment:
Cool! I like that you did this purely with CSS since it's just presentational and doesn't include any actual content.

Reply:
Thanks! I didn't know about the 'before' tags prior to this so it was interesting to see how they worked!